authors/apps/authentication/serializers.py

```diff
@@ -8,13 +9,40 @@
 class RegistrationSerializer(serializers.ModelSerializer):
     """Serializers registration requests and creates a new user."""
 
-    # Ensure passwords are at least 8 characters long, no longer than 128
+    # Ensure passwords are at least 8 characters long,contain alphanumerics
+    # special characters and
+    # no longer than 128
     # characters, and can not be read by the client.
-    password = serializers.CharField(
+    password = serializers.RegexField(
+        regex='^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[@#$%^&+=*!])',
         max_length=128,
         min_length=8,
-        write_only=True
+        write_only=True,
+        error_messages={
+            'required': 'Password field required',
+            'min_length': 'Ensure Password field has at least 8 characters',
+            'invalid': 'password should contain a lowercase, uppercase numeric'
```

Comment:
Start error message with capital letter just like the others.

Reply:
Okay